data/survey/6002/meta.json

```diff
@@ -0,0 +1,79 @@
+[
+    {
+        "type": "CHOICE",
+        "question": "Do you feel more knowledgable about computer networking than before you took this module?",
+        "options": [
+            "Yes",
+            "About the same",
+            "No"
+        ]
+    },
+    {
+        "type": "SCALE",
+        "question": "How helpful were the diagrams in understanding protocols?",
+        "options": [
+            "Not at all",
+            "A little",
+            "Not very",
+            "Super helpful"
+        ]
+    },
+
+    {
+        "type": "SCALE",
+        "question": "How helpful was the example of chain of command in understanding OSI layers?",
+        "options": [
+            "Not at all",
+            "A little",
+            "Not very",
+            "Super helpful"
+        ]
+    },
+    {
+        "type": "TEXT",
+        "question": "How difficult as the material to master?"
+    },
+    {
+        "type": "LONGTEXT",
+        "question": "Please list any questions you still have about the content."
+    }
+][
```

Comment:
Like Blaze mentioned, this looks like a type. I tried to parse this file as JSON as well and it threw an error 
> VM229:40 Uncaught SyntaxError: Unexpected token [ in JSON at position 998
    at JSON.parse (<anonymous>)
    at <anonymous>:1:6